client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -3,24 +3,77 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { localize } from 'i18n-calypso';
-
+import { map } from 'lodash/map';
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
 
 class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
+	state = {
+		city: '',
+		name: '',
+		stateName: '',
+		street: '',
+		zip: '',
+	};
+
+	getChangeHandler = field => event => {
+		this.setState( { [ field ]: event.target.value } );
+	};
+
 	render() {
 		const { translate } = this.props;
 		const headerText = translate( 'Add a business address.' );
 		const subHeaderText = translate(
 			'Enter your business address to have a map added to your website.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		const fields = getFields( translate );
+
+		return (
+			<Fragment>
+				<DocumentHead title={ translate( 'Business Address ‹ Jetpack Onboarding' ) } />
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+
+				<Card className="steps__form">
+					<form>
+						{ Object.keys( fields ).map( field => (
+							<FormFieldset key={ field }>
+								<FormLabel htmlFor={ field }>{ fields[ field ] }</FormLabel>
+								<FormTextInput
+									id={ field }
+									onChange={ this.getChangeHandler( field ) }
+									value={ this.state[ field ] }
+									autoFocus={ field === 'name' }
+								/>
+							</FormFieldset>
+						) ) }
+						<Button href={ this.props.getForwardUrl() } primary>
+							{ translate( 'Next Step' ) }
+						</Button>
+					</form>
+				</Card>
+			</Fragment>
+		);
 	}
 }
 
+function getFields( translate ) {
+	return {
+		name: translate( 'Business Name' ),
+		street: translate( 'Street Address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Invalid street address' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-site-business-address)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).